app/frontend/covid/covid.js

```diff
@@ -88,6 +88,25 @@ const Covid = {
     $('#modal-icon').html(iconHTML);
 
     $(modal).attr('x-data', '{ open: true }');
+  },
+  skillBadges(items, color = 'indigo', title = '') {
+    let classes = ''
+    if (color == 'blue') {
```

Comment:
we should add a linter, but please add ; at the end

Reply:
Will do